fix(cascade): registry.set() on ColorValue family does not cascade contrast:auto dependents

## Context

Integration test: `accent-foreground cascades to a new value after accent colorWheel update` (color-wheel-integration.test.ts)

## What fails

When `registry.set('accent', newColorValue)` is called with a `ColorValue` object (as produced by `colorWheel()`), dependents whose `generationRule` is `contrast:auto` (e.g. `accent-foreground`) do not regenerate. After the set, `accent-foreground` still returns `{"family":"neutral","position":"900"}` -- the pre-update value.

## Error

```
AssertionError: accent-foreground did not cascade after family change:
  expected '{"family":"neutral","position":"900"}'
  not to be '{"family":"neutral","position":"900"}'
```

## Root cause hypothesis

The dependency graph registers `accent-foreground` as depending on `accent` (the family name). When `registry.set()` receives a full `ColorValue` as the new value and calls `regenerateDependents('accent')`, the `scale:N` plugin attempts to extract a scale position from the token name `accent` (not a position-qualified name like `accent-500`). This throws `"Cannot extract scale position from token name: accent"` which is swallowed by the `try/catch` in `regenerateDependents`, silently preventing the `contrast:auto` cascade from running.

The stderr output during tests confirms: `Failed to regenerate token primary: Error: Cannot extract scale position from token name: primary`.

## Steps to reproduce

```ts
const registry = await buildRegistry(TAILWIND_BLUE_500); // see color-wheel-integration.test.ts
const fgBefore = registry.get('accent-foreground')?.value;
const newSystem = colorWheel(BRAND_MAGENTA, 'complementary');
await registry.set('accent', newSystem.accent);
const fgAfter = registry.get('accent-foreground')?.value;
// fgBefore === fgAfter -- cascade did not fire
```

## Fix scope

`packages/design-tokens/src/registry.ts` -- `regenerateDependents` and/or `regenerateToken`. When the changed token is a family-level ColorValue (no position suffix), the cascade should iterate dependents and run their own rules against the new ColorValue. The scale:N error on the family token itself should not abort the entire dependent chain.

## Linked test

Closes part of #1221 acceptance criteria.

--- a/packages/design-tokens/src/registry.ts
+++ b/packages/design-tokens/src/registry.ts
@@ -565,12 +565,10 @@ export class TokenRegistry {
 
     if (typeof resolvedValue === 'object' && 'family' in resolvedValue) {
       const ref = resolvedValue as ColorReference;
-      const existingIsRef =
-        typeof existingToken.value === 'object' &&
-        existingToken.value !== null &&
-        'family' in existingToken.value;
+      // Semantic tokens store ColorReferences; position tokens store CSS strings.
+      const isSemanticToken = existingToken.namespace === 'semantic';
 
-      if (existingIsRef) {
+      if (isSemanticToken) {
         // Semantic token path: update dependsOn[0]=family, dependsOn[1]=dark counterpart.
         // findDarkCounterpart requires WCAG data from the family ColorValue.
         const darkTokenName = this.findDarkCounterpart(ref, existingToken.dependsOn);

